docs/inventory.md

```diff
@@ -405,6 +405,20 @@ The basic network configuration of the inventory for the fully bare metal deploy
           bmc_address: 172.30.10.7
           # ...
 ```
+## Additional Partition Deployment
+
+Uncomment the lines below in the inventory.yml to set additional partitions if required on the nodes. Please provide the correct label and size(MiB).
+Add all additional paritions required under extra_partitions (examples for label:size provided below).
+Additional partitions can only be created for OCP 4.8+ deployments
+
+```yaml
+# ...
+disks:
+  - device: "{{ installation_path }}" #DO NOT CHANGE
```

Comment:
You might want to change that for instance if you have a second disk you wish to format